### PR TITLE
Removing learn more based on hostgatro brand

### PR DESCRIPTION
**Pull request by ramyakrishnai**
from bluehost
Screenshot 2023-09-04 at 6 01 41 PM

from hostgator
Screenshot 2023-09-04 at 6 01 18 PM

### CODE REVIEW

File: includes/Data/Brands.php [hunk 1]
```diff
@@ -53,6 +53,23 @@ public static function get_config( Container $container ) {
...
+					'url' => 'https://hostgator.com',
+					'hireExpertsInfo' => 'admin.php?page=hostgator#/marketplace/services/blue-sky',
+					'support' => 'https://www.hostgator.com/contact',
+					'adminPage' => 'admin.php?page=hostgator',
+					'setup' => array(
+						'payment' => array('Paypal', 'Razorpay'),
+						'shipping' => array('Shippo'),
+					),
+					'defaultContact' => array(
+						'woocommerce_default_country' => 'US:AZ',
```
Comment (aulisius) [inline includes/Data/Brands.php:69]: Are we sure these values are correct for Hostgator?

File: includes/Data/Brands.php [hunk 2]
```diff
@@ -53,6 +53,23 @@ public static function get_config( Container $container ) {
...
+			case 'hostgator':
+				return array(
+					'brand' => 'hostgator',
+					'name' => 'hostgator',
+					'url' => 'https://hostgator.com',
+					'hireExpertsInfo' => 'admin.php?page=hostgator#/marketplace/services/blue-sky',
+					'support' => 'https://www.hostgator.com/contact',
+					'adminPage' => 'admin.php?page=hostgator',
+					'setup' => array(
+						'payment' => array('Paypal', 'Razorpay'),
```
Comment (aulisius) [inline includes/Data/Brands.php:65]: Do we need Razorpay for HG?

File: src/configs/selectors.js [hunk 3]
```diff
@@ -9,6 +9,19 @@ export const yithOnboardingParser = (title) => (data) => {
   return { isCompleted: data?.[title] == "true" };
 };
 
+export const yithOnboardingPaymentParser = (titles) => (data) => {
```
Comment (aulisius) [inline src/configs/selectors.js:12]: Can be simplified as 
```js
let isCompleted = titles.some(key => data?.[key] === "true");
```

**Review by aulisius — CHANGES_REQUESTED** [review]

File: includes/Data/Brands.php (hunk 2 above)
Comment (ramyakrishnai) [reply includes/Data/Brands.php:65]: yes, we need both

**Review by aratidgr8 — APPROVED** [review]

**Review by aulisius — APPROVED** [review]

**Pull request merged**